packages/runtime/src/lib/txn.ts

```diff
@@ -52,7 +52,8 @@ export function txnSpecbyField (txField: string, tx: Txn, gtxns: Txn[], tealVers
   if (assetTxnFields.has(txField)) {
     const s = TxnFields[tealVersion][txField];
     const assetMetaData = tx.apar;
-    result = assetMetaData[s as keyof AssetDefEnc];
+    if (assetMetaData === undefined) result = undefined;
+    else result = assetMetaData[s as keyof AssetDefEnc];
```

Comment:
I am only returning the `GlobalZeroAddress` here. I couldn't find anything related to this bug on Algorand Docs.

Reply:
I tried running the similar transactions on the network and since it also processes this property as undefined so it must be returning something(probably undefined or GlobalZeroAddress(Which is the default value I'm returning here)) to make the transaction not fail.

Reply:
I think we need to check if this is verified in the docs or not.
Regarding the implementation, can we just do:
```js
result = assetMetaData?[s as keyof AssetDefEnc];
```
?

Reply:
I'll change the implementation, but I didn't find anything related to this on Algorand docs.